WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1695771353.697865  196953 tfrt_cpu_pjrt_client.cc:349] TfrtCpuClient created.
No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/scvi/_settings.py:63: UserWarning: Since v1.0.0, scvi-tools no longer uses a random seed by default. Run `scvi.settings.seed = 0` to reproduce results from previous versions.
  self.seed = seed
/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/scvi/_settings.py:70: UserWarning: Setting `dl_pin_memory_gpu_training` is deprecated in v1.0 and will be removed in v1.1. Please pass in `pin_memory` to the data loaders instead.
  self.dl_pin_memory_gpu_training = (
Traceback (most recent call last):
  File "/home/icb/alessandro.palma/environment/scCFM/scCFM/train_hydra/utils/exceptions.py", line 11, in __call__
    return self.f(*args, **kwargs)
  File "/home/icb/alessandro.palma/environment/scCFM/scCFM/train_hydra/train_cfm.py", line 94, in main
    logger = WandbLogger(save_dir=current_experiment_dir,
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/pytorch_lightning/loggers/wandb.py", line 360, in __init__
    _ = self.experiment
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/lightning_fabric/loggers/logger.py", line 118, in experiment
    return fn(self)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/pytorch_lightning/loggers/wandb.py", line 408, in experiment
    self._experiment = wandb.init(**self._wandb_init)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1184, in init
    raise e
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1161, in init
    wi.setup(kwargs)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 189, in setup
    self._wl = wandb_setup.setup(settings=setup_settings)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 327, in setup
    ret = _setup(settings=settings)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 320, in _setup
    wl = _WandbSetup(settings=settings)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 303, in __init__
    _WandbSetup._instance = _WandbSetup__WandbSetup(settings=settings, pid=pid)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 114, in __init__
    self._setup()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 250, in _setup
    self._setup_manager()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 277, in _setup_manager
    self._manager = wandb_manager._Manager(settings=self._settings)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_manager.py", line 138, in __init__
    self._service.start()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/service.py", line 245, in start
    self._launch_server()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/service.py", line 239, in _launch_server
    _sentry.reraise(e)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/analytics/sentry.py", line 154, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/service.py", line 237, in _launch_server
    self._wait_for_ports(fname, proc=internal_proc)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/service.py", line 130, in _wait_for_ports
    raise ServiceStartTimeoutError(
wandb.sdk.service.service.ServiceStartTimeoutError: Timed out waiting for wandb service to start after 30.0 seconds. Try increasing the timeout with the `_service_wait` setting.
Error executing job with overrides: ['datamodule=CFM_pancreas_flat_high', 'logger=CFM_pancreas_flat_high', 'train=CFM_pancreas_flat_high']
Traceback (most recent call last):
  File "/home/icb/alessandro.palma/environment/scCFM/scCFM/train_hydra/train_cfm.py", line 125, in <module>
    main()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/hydra/main.py", line 94, in decorated_main
    _run_hydra(
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/hydra/_internal/utils.py", line 394, in _run_hydra
    _run_app(
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/hydra/_internal/utils.py", line 457, in _run_app
    run_and_report(
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/hydra/_internal/utils.py", line 223, in run_and_report
    raise ex
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/hydra/_internal/utils.py", line 220, in run_and_report
    return func()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/hydra/_internal/utils.py", line 458, in <lambda>
    lambda: hydra.run(
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/hydra/_internal/hydra.py", line 132, in run
    _ = ret.return_value
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/hydra/core/utils.py", line 260, in return_value
    raise self._return_value
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/hydra/core/utils.py", line 186, in run_job
    ret.return_value = task_function(task_cfg)
  File "/home/icb/alessandro.palma/environment/scCFM/scCFM/train_hydra/utils/exceptions.py", line 11, in __call__
    return self.f(*args, **kwargs)
  File "/home/icb/alessandro.palma/environment/scCFM/scCFM/train_hydra/train_cfm.py", line 94, in main
    logger = WandbLogger(save_dir=current_experiment_dir, 
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/pytorch_lightning/loggers/wandb.py", line 360, in __init__
    _ = self.experiment
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/lightning_fabric/loggers/logger.py", line 118, in experiment
    return fn(self)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/pytorch_lightning/loggers/wandb.py", line 408, in experiment
    self._experiment = wandb.init(**self._wandb_init)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1184, in init
    raise e
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1161, in init
    wi.setup(kwargs)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 189, in setup
    self._wl = wandb_setup.setup(settings=setup_settings)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 327, in setup
    ret = _setup(settings=settings)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 320, in _setup
    wl = _WandbSetup(settings=settings)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 303, in __init__
    _WandbSetup._instance = _WandbSetup__WandbSetup(settings=settings, pid=pid)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 114, in __init__
    self._setup()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 250, in _setup
    self._setup_manager()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_setup.py", line 277, in _setup_manager
    self._manager = wandb_manager._Manager(settings=self._settings)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/wandb_manager.py", line 138, in __init__
    self._service.start()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/service.py", line 245, in start
    self._launch_server()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/service.py", line 239, in _launch_server
    _sentry.reraise(e)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/analytics/sentry.py", line 154, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/service.py", line 237, in _launch_server
    self._wait_for_ports(fname, proc=internal_proc)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/service.py", line 130, in _wait_for_ports
    raise ServiceStartTimeoutError(
wandb.sdk.service.service.ServiceStartTimeoutError: Timed out waiting for wandb service to start after 30.0 seconds. Try increasing the timeout with the `_service_wait` setting.
Traceback (most recent call last):
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/__main__.py", line 3, in <module>
    cli.cli(prog_name="python -m wandb")
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/cli/cli.py", line 102, in wrapper
    return func(*args, **kwargs)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/cli/cli.py", line 279, in service
    server.serve()
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/server.py", line 114, in serve
    self._inform_used_ports(sock_port=sock_port)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/server.py", line 52, in _inform_used_ports
    pf.write(self._port_fname)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/site-packages/wandb/sdk/service/port_file.py", line 21, in write
    f = tempfile.NamedTemporaryFile(prefix=bname, dir=dname, mode="w", delete=False)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/tempfile.py", line 559, in NamedTemporaryFile
    file = _io.open(dir, mode, buffering=buffering,
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/tempfile.py", line 556, in opener
    fd, name = _mkstemp_inner(dir, prefix, suffix, flags, output_type)
  File "/home/icb/alessandro.palma/miniconda3/envs/scCFM/lib/python3.10/tempfile.py", line 256, in _mkstemp_inner
    fd = _os.open(file, flags, 0o600)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpwq7i22a8/port-196953.txto40h8bk0'
